devman

```diff
@@ -218,7 +218,7 @@ class DevMan:
             except Exception as e:
               print "Can't clone {0}: {1}".format(repo_name, e)
           else:
-            print 'Error: Repository %s does not exist. Please use git clone first.' % repo_name
+            print 'Error: Repository %s doesn\'t exist. Please use git clone first.' % repo_name
```

Comment:
For readability, use triple quotes instead of escapes:
```
weisheit :: ~ % python                                                                                              
Python 2.7.12rc1 (default, Jun 13 2016, 09:20:59) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> repo_name="test"
>>> print """Error: Repository %s doesn't exist. Please use git clone first.""" % repo_name
Error: Repository test doesn't exist. Please use git clone first.
>>> 
```